packages/account/src/Components/poi/poi-country-selector/poi-country-selector.tsx

```diff
@@ -78,9 +80,30 @@ const CountrySelector = ({
                             'min-height': !is_from_external,
                         })}
                     >
-                        <Text className='proof-of-identity__header' align='center' weight='bold'>
-                            {localize('Proof of identity')}
-                        </Text>
+                        {is_verification_failed ? (
+                            <React.Fragment>
+                                <Text className='' align='center' weight='bold'>
+                                    {localize('Your identity verification failed because:')}
+                                </Text>
+                                <HintBox
+                                    icon='IcCloseCircleRed'
+                                    icon_height={16}
+                                    icon_width={16}
+                                    message={
+                                        <Text as='p' size='xs'>
+                                            {localize('Your identity document has expired.')}
+                                        </Text>
```

Comment:
Are we showing the same expired message for all verification failed cases?

Reply:
thanks for noticing it, will check and fix